tmpback/components/youtube/youtube.html

```diff
@@ -0,0 +1,92 @@
+<link rel="import" href="/bower_components/google-youtube/google-youtube.html">
+<dom-module id="youtube-item">
+  <template>
+    <style>
+      :host {
+        display: none;
+        position: absolute;
+        top: 0px;
+        right: 0px;
+        left: 0px;
+        bottom: 0px;
+        background-color: black;
+      }
+    </style>
+    <google-youtube id="googleYoutube" video-id="[[video_id]]" autoplay="1" width="100%" height="100%" controls="0"></google-youtube>
+  </template>
+  <script>
+    (() => {
+      class Youtube extends Polymer.Element {
+        static get is() { return 'youtube-item' }
+
+        static get properties() {
+          return {
+            title: String,
+            video_id: String
+          }
+        }
+
+        connectedCallback() {
+          super.connectedCallback();
+          socket.on('youtube', (request) => this._handleRequest(request));
+        }
+
+        _handleRequest (req) {
+          switch (req.type) {
+            case 'search':
+              this._searchVideo(req.content);
+              break;
```

Comment:
si tu fait `return this._seatchVideo(req.content);` tu peux te débarasser des `break` 🙂 Mais c'est comme tu préfère !